assets/chezmoi.io/mkdocs.yml

```
@@ -6,6 +6,7 @@ copyright: Copyright © Tom Payne 2018-2024
 repo_name: twpayne/chezmoi
 repo_url: https://github.com/twpayne/chezmoi
 edit_uri: edit/master/assets/chezmoi.io/docs/
+strict: true
```

Let's add more strictness and also enable validation for links with anchors and link in invalid format. See recommended settings in https://www.mkdocs.org/user-guide/configuration/#validation
```suggestion
strict: true

validation:
  omitted_files: warn
  absolute_links: relative_to_docs
  unrecognized_links: warn
  anchors: warn
```